switchlink/switchlink_db.c

```diff
@@ -318,7 +318,7 @@ switchlink_db_status_t switchlink_db_tunnel_interface_get_info(
  *    SWITCHLINK_DB_STATUS_ITEM_NOT_FOUND otherwise
  */
 
-switchlink_db_status_t switchlink_db_tunnel_interface_delete(uint32_t ifindex) {
+switchlink_db_status_t switchlink_db_tunnel_delete_interface(uint32_t ifindex) {
```

Comment:
The object being deleted is a _tunnel _interface__, so this should be `..._delete_tunnel_interface`.